go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -638,6 +638,7 @@ func (npw *nodePortWatcher) SyncServices(services []interface{}) {
 		}
 		recreateIPTRules("mangle", iptableITPChain, keepIPTRules)
```

Comment:
so this function needs a big TODO to fix propagating errors upward. Either you can fix it in this PR or make a github issue and we fix it later. Calls in this function that should produce errors are being ignored. For example recreateIPTables simply logs errors and instea dit needs to return an error type. npw.updateServiceFlowCache is the same thing.

Reply:
Tracked under https://bugzilla.redhat.com/show_bug.cgi?id=2081857